### PR TITLE
Base image + .gitignore + Travis

### PR DESCRIPTION
**Pull request by ofir-petrushka**
Reuse docker travis tricks from github.com/lukecyca/travis-docker-example, now with returned error codes to actually fail travis (using peru + makefile + .travis..)
.travis.yml ready for multi image tests in parallel
Base docker images.
Initial .gitignore

@rounds/server

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.peru
+.swp
```
Comment (oryband) [inline .gitignore]: what is `.swp` for?

File: base/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Base 10M docker
+===============
+All other round dockers should be built on top of it.
+
+Changes here will cause all images to rebuild.
```
Comment (oryband) [inline base/README.md]: I didn't understand anything from this readme.

**Comment by oryband** [comment]
looks cool
commit message was a bit confusing, but nm

File: .gitignore (hunk 1 above)
Comment (ofir-petrushka) [inline .gitignore]: .peru - Peru cache directories when fetching files.
.swp - vim swap files.
.SyncOld - btsync files.

Comment (oryband) [inline .gitignore]: so it should be `*.swp` i think

File: base/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Base 10M docker
+===============
+This is the master branch of the tree, all other images inherit from it either directly or by casscade inheritance.
```
Comment (oryband) [inline base/README.md]: cascade

File: base/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
+Any change here should cause a casscase rebuild of the entire tree.
```
Comment (oryband) [inline base/README.md]: --^

File: uml.sh [hunk 5]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```
Comment (oryband) [inline uml.sh]: this is not an issue at all, but i always prefer `#!/bin/sh` (old shell compliant) if not explicitly using bash syntax, which you are't in this case

Comment (oryband) [inline uml.sh]: (i think, right?)

Comment (ofir-petrushka) [inline uml.sh]: This is mostly pull from another repo, see peru.yaml.
So I try to make the least changes possible which currently is what you can see in the makefile +
Adding line 10 which I should also somehow put into the makefile.

File: base/README.md (hunk 4 above)
Comment (ofir-petrushka) [inline base/README.md]: 10x, fixed.

**Pull request merged**